executor/show.go

```diff
@@ -694,3 +697,91 @@ func (e *ShowExec) versionToTime(version uint64) types.Time {
 	t := time.Unix(0, oracle.ExtractPhysical(version)*int64(time.Millisecond))
 	return types.Time{Time: types.FromGoTime(t), Type: mysql.TypeDatetime}
 }
+
+func (e *ShowExec) fetchShowStatsBuckets() error {
+	do := sessionctx.GetDomain(e.ctx)
+	h := do.StatsHandle()
+	dbs := do.InfoSchema().AllSchemas()
```

Comment:
I didn't find a way to get the `db_name` from a `tableId`, so I implement it this way.